tests/test_errors.py

```diff
@@ -405,6 +405,18 @@ def test_handle_smart_errors(self, app):
             'message': NotFound.description
         }
 
+    def test_handle_not_include_error_message(self, app):
+        app.config['ERROR_INCLUDE_MESSAGE'] = False
+
+        api = restplus.Api(app)
+        view = restplus.Resource
+
+        api.add_resource(view, '/foo', endpoint='bor')
+
+        with app.test_request_context("/faaaaa"):
+            response = api.handle_error(NotFound())
+            assert 'message' not in json.loads(response.data.decode())
```

Comment:
I would add another test for the use case described above

Reply:
Added additional test to check message in response